src/plugins/files/common/base_file_kind.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { FileKind } from './types';
+
+const id = 'baseFileKind' as const;
```

Comment:
nit: maybe simply `default`? "base" sounds like something that can be *extended*. And "FileKind" suffix specifies the type of string, like in [Hungarian Notation](https://en.wikipedia.org/wiki/Hungarian_notation), but sort of redundant.

```suggestion
const id = 'default' as const;
```

Reply:
Wow, TIL Hungarian notation